packages/reports/addon/templates/reports/report/edit.hbs

```diff
@@ -8,7 +8,7 @@
   {{#with (model-for (parent-route)) as | report |}}
     <p>
       Select some dimensions and metrics, then click
-      <a class="navi-link clickable"
+      <a href class="navi-link clickable"
```

Comment:
`{{!-- template-lint-disable link-href-attributes --}}`?

We should eventually convert all these non `href` `<a>` elements to something else.

Reply:
Good catch, idk how I messed this one up. Yea I'll make an issue for this so that it we can track it